src/bootchart.c

```diff
@@ -351,6 +352,10 @@ int main(int argc, char *argv[]) {
         rlim.rlim_max = 4096;
         (void) setrlimit(RLIMIT_NOFILE, &rlim);
 
+        schfd = open("/proc/sys/kernel/sched_schedstats", O_WRONLY);
+        if (schfd)
```

Comment:
Isn't it better to close this file immediately? Keeping the fd open until the end of main seems a bit wasteful…